File: ai/ai.php
<?php
$msg = $_GET["msg"] ?? implode(' ', array_slice($_SERVER['argv'], 1)) ?? null;
if (empty($msg)) {
    echo "[*]Usage: php ai.php [question]\n";
    echo "[*]使用方法: php ai.php [你的问题]\n";
    exit;
}
$msg = urlencode($msg);
$url = "https://api.52vmy.cn/api/chat/glm?msg={$msg}&type=text";
$result = file_get_contents($url);
echo "\n===== AI Chat =====\n$result\n\n";
?>

File: ai/ai.py
import sys
import urllib.parse
import urllib.request

def main():
    if len(sys.argv) < 2:
        print("[*]Usage: python ai.py [question]")
        print("[*]用法：python ai.py [你的问题]")
        return
    msg = " ".join(sys.argv[1:])

    msg_enc = urllib.parse.quote(msg)
    url = f"https://api.52vmy.cn/api/chat/glm?msg={msg_enc}&type=text"

    try:
        req = urllib.request.Request(url)
        with urllib.request.urlopen(req) as res:
            result = res.read().decode("utf-8")
            print("\n===== AI Chat =====\n" + result + "\n")
    except Exception as e:
        print(f"[!]请求失败：{e}")

if __name__ == "__main__":
    main()
